.gitignore

```diff
@@ -16,7 +16,8 @@ doc-tools
 release
 misc/*
 src/release.h
-appendonly.aof
+appendonly.aof.*
```

Comment:
should be `appendonly.aof*` instead of `appendonly.aof.*`  
now, `appendonly.aof` will not be ignored.

Reply:
Maybe you're right, but the current implementation doesn't create a file named `appendonly.aof` unless you manually copy an old-style AOF for upgrade testing. So I think we just need to ignore the file we're going to create. @oranagra WDYT?

Reply:
i think it's a good idea to ignore both.
i.e. both the ones we create, and the ones we created in the past, that are still left in the folder.

Reply:
Only developers pay attention to `.gitignore`, and when I updated the `unstable brance` code, `appendonly.aof` caught my attention.

Reply:
@oranagra @yangbodong22011  Sounds reasonable, I'll make a PR to add it, thanks.